doc/user/zebra.rst

```diff
@@ -1271,6 +1271,15 @@ with the linux kernel for that.
    that used that interface being removed from the FIB.  This option is recommended because this
    is the default behavior for v4 as well.
 
+Useful sysctl settings
+======================
+
+.. option:: net.ipv6.conf.all.use_oif_addrs_only=1
+
+   When enabled, the candidate source addresses for destinations routed via this interface are
+   restricted to the set of addresses configured on this interface (RFC 6724 section 4)
```

Comment:
It might be useful to mention that if you have hundreds of IP addresses per interface (especially for IPv6) this solves the performance (latency) problem. 

P.S. dot is missing the the end :)